tests/pyoozie/test_builder.py

```diff
@@ -130,7 +119,39 @@ def test_coordinator_submission_xml_with_configuration(username, coord_app_path)
     </configuration>''') == tests.utils.xml_to_dict_unordered(actual)
 
 
-def test_workflow_builder(tmpdir, workflow_builder):
+def test_workflow_builder(tmpdir):
+
+    workflow_builder = builder.WorkflowBuilder(
+        name='descriptive-name',
+        job_tracker='job-tracker',
+        name_node='name-node',
+    ).add_action(
+        name='extract',
+        action=tags.Shell(exec_command='echo', arguments=["'Extract data from an operational system'"]),
+        action_on_error=tags.Email(to='person@example.com', subject='Error',
```

Comment:
I'd like to keep users as oblivious to names as I can; if you wanted to share the same action on error between actions we could give the same action_on_error instance to multiple actions.